modules/CurveAnalysis/src/server/renderCurve.R

```diff
@@ -25,67 +25,16 @@ renderCurve <- function(getParams, postData) {
   # GET FIT DATA
   fitData <- racas::get_curve_data(parsedParams$curveIds, raw_data = TRUE, globalConnect = TRUE)
   
-  # Colors
```

Comment:
This all got moved to an racas function so that we could reuse it in the dose response validation